src/Symfony/Component/Form/Resources/translations/validators.th.xlf

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="th" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="28">
+                <source>This form should not contain extra fields.</source>
+                <target>ฟอร์มนี้ไม่ควรมี extra fields</target>
+            </trans-unit>
+            <trans-unit id="29">
+                <source>The uploaded file was too large. Please try to upload a smaller file.</source>
+                <target>ไฟล์ที่อัพโหลดมีขนาดใหญ่เกินไป กรุณาลองอัพโหลดใหม่อีกครั้งด้วยไฟล์ที่มีขนาดเล็กลง</target>
+            </trans-unit>
+            <trans-unit id="30">
+                <source>The CSRF token is invalid. Please try to resubmit the form.</source>
+                <target>CSRF token ไม่ถูกต้อง กรุณาลองส่งแบบฟอร์มใหม่</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>ค่านี้ไม่ใช่ค่าที่ถูกต้องของค่าสี HTML5</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>กรุณากรอกวันเดือนปีเกิดที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>ตัวเลือกที่เลิอกไม่ถูกต้อง</target>
```

Comment:
Can you double check if this is correct?

Reply:
It got the result "ตัวเลือกที่เลือกไม่ถูกต้อง" from Google Translate.
It same as I translated.

Reply:
Google translate might be wrong. I just wanted you to confirm that this was correct. Thank you